includes/modules/bootstrap/category_row.php

```diff
@@ -17,18 +17,19 @@
 }
 
 $num_categories = $categories->RecordCount();
+if (empty($num_categories)) {
+    return;
+}
 
 $rows = 0;
 $columns = 0;
 $list_box_contents = [];
 $title = '';
 
-if (empty($num_categories)) {
-    return;
-}
-
 $columns_per_row = defined('MAX_DISPLAY_CATEGORIES_PER_ROW') ? (int)MAX_DISPLAY_CATEGORIES_PER_ROW : 0;
-$category_row_layout_style = $columns_per_row > 1 ? 'columns' : 'fluid';
+if (empty($category_row_layout_style) || !in_array($category_row_layout_style, ['columns', 'table', 'fluid'])) {
```

Comment:
FWIW, the categories' row has never (before) been rendered as a 'table'.  What's the reasoning for this change, other than to align with the products-listing layout?

Reply:
Right. `table` can be removed. I'll update it.

Reply:
It's more than that, having all those soft-overrides for the template has the possibility of making the bootstrap template's support of the categories' row-layout that much more difficult.  I see no layout issues with how it's been released for v3.6.3 of the template.

Reply:
Variables renamed for less chance of clash between each other.